[2023-12-27T03:54:17.681+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_customers_table manual__2023-12-27T03:54:06.967094+00:00 [queued]>
[2023-12-27T03:54:17.786+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: potential_customers_process.create_customers_table manual__2023-12-27T03:54:06.967094+00:00 [queued]>
[2023-12-27T03:54:17.788+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-12-27T03:54:17.791+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-12-27T03:54:17.791+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-12-27T03:54:17.868+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_customers_table> on 2023-12-27 03:54:06.967094+00:00
[2023-12-27T03:54:17.972+0000] {standard_task_runner.py:55} INFO - Started process 126 to run task
[2023-12-27T03:54:18.025+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'potential_customers_process', 'create_customers_table', 'manual__2023-12-27T03:54:06.967094+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/potential_customers_process.py', '--cfg-path', '/tmp/tmps1fnol6s']
[2023-12-27T03:54:18.033+0000] {standard_task_runner.py:83} INFO - Job 14: Subtask create_customers_table
[2023-12-27T03:54:18.425+0000] {task_command.py:376} INFO - Running <TaskInstance: potential_customers_process.create_customers_table manual__2023-12-27T03:54:06.967094+00:00 [running]> on host bbee0f1e1252
[2023-12-27T03:54:18.747+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=potential_customers_process
AIRFLOW_CTX_TASK_ID=create_customers_table
AIRFLOW_CTX_EXECUTION_DATE=2023-12-27T03:54:06.967094+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-27T03:54:06.967094+00:00
[2023-12-27T03:54:18.804+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-12-27T03:54:18.828+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS customers (
                fullname TEXT NOT NULL,
                country_code TEXT NOT NULL,
                gender TEXT NOT NULL,
                age INT NOT NULL,
                last_login INT NOT NULL
            );
        , parameters: None
[2023-12-27T03:54:18.834+0000] {postgres.py:96} INFO - NOTICE:  relation "customers" already exists, skipping

[2023-12-27T03:54:18.874+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=potential_customers_process, task_id=create_customers_table, execution_date=20231227T035406, start_date=20231227T035417, end_date=20231227T035418
[2023-12-27T03:54:19.061+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-12-27T03:54:19.427+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
